ext/CEGUI/CMakeLists.txt

```diff
@@ -1,26 +1,42 @@
 include(../../cmake/common.cmake)
 update_git_submodule(./stb)
 
-
 set(IRR_EXT_CEGUI_H
   ExtCEGUI.h
 	CEGUIOpenGLState.h
 	CEGUIOpenGLClip.h
 	Helpers.h
+	IrrlichtBaW/IrrlichtBaWRenderer.hpp
+	IrrlichtBaW/IrrlichtBaWRenderTarget.hpp
+	IrrlichtBaW/IrrlichtBaWRenderQueue.hpp
+	IrrlichtBaW/IrrlichtBaWGeometryBuffer.hpp
+	IrrlichtBaW/IrrlichtBaWRenderingWindow.hpp
+	IrrlichtBaW/IrrlichtBaWRenderEffect.hpp
+	IrrlichtBaW/IrrlichtBaWTextureTarget.hpp
+	IrrlichtBaW/IrrlichtBaWTexture.hpp
```

Comment:
I wouldn't bother with the extra `IrrlichtBaW` directory since you already prefix the file/class names

Reply:
resolved